docs/setup/config.md

```diff
@@ -389,6 +389,10 @@ These parameters allow for advanced tuning. The default values are sufficient wh
 
 - `jobmanager.web.access-control-allow-origin`: Enable custom access control parameter for allow origin header, default is `*`.
 
+### Web Frontend
```

Comment:
I think that having some options tied to the jobmanager while others aren't will just cause more work down the line. Either add it the the jobmanager options now and move it in a follow-up, or separate the options now and rebase the PR on top of it.

The "let's refactor the rest later" mentality just doesn't work for us, see the translation to ConfigOptions which _still isn't done_.